### DISCUSSION TITLE
Error when updating to Paste 19

### DISCUSSION THREAD

**Discussion by mandrioli-twilio** [twilio-labs/paste#3357] [General] [answered]
Hey team, I'm working on a project using Paste 18, turborepo and pnpm. I'm trying to update to Paste 19 but I get the following error on pnpm-lock.yaml file. 

`WARNING  Issues occurred when constructing package graph. Turbo will function, but some features may not be available:  1 error occurred:
	* error performing lockfile operation: Unable to convert to yaml: could not find expected ':' at line 6616 column 1031, while scanning a simple key at line 6616 column 3`

This is the line 6616:
`
  /@twilio-paste/combobox@15.0.3(@twilio-paste/anchor@11.0.0)(@twilio-paste/animation-library@1.0.0)(@twilio-paste/box@9.2.0)(@twilio-paste/button@13.0.3)(@twilio-paste/color-contrast-utils@4.0.0)(@twilio-paste/combobox-primitive@1.0.1)(@twilio-paste/customization@7.0.0)(@twilio-paste/design-tokens@9.2.1)(@twilio-paste/dropdown-library@2.0.0)(@twilio-paste/flex@7.0.0)(@twilio-paste/form-pill-group@7.0.0)(@twilio-paste/help-text@12.0.1)(@twilio-paste/icons@11.2.2)(@twilio-paste/input-box@9.0.1)(@twilio-paste/input@8.1.0)(@twilio-paste/label@12.0.2)(@twilio-paste/media-object@9.0.0)(@twilio-paste/reakit-library@1.0.0)(@twilio-paste/screen-reader-only@12.0.0)(@twilio-paste/spinner@13.0.0)(@twilio-paste/stack@7.0.0)(@twilio-paste/style-props@8.0.1)(@twilio-paste/styling-library@2.0.2)(@twilio-paste/text@9.0.3)(@twilio-paste/theme@10.0.3)(@twilio-paste/types@5.0.0)(@twilio-paste/uid-library@1.0.0)(@twilio-paste/utils@4.1.0)(@types/react-dom@18.2.7)(@types/react@18.2.17)(prop-types@15.7.2)(react-dom@18.0.0)(react@18.0.0):
`

and package.json:

```
"@twilio-paste/core": "^19.6.0",
"@twilio-paste/icons": "^11.2.2",
```

I've found out this happens after a commit, when the project runs `pretty-quick`, is this something related to Paste or something else?

Thanks

**Comment by loreina** [comment]
Hi @mandrioli-twilio,

Can you share a minimal reproduction code repository so we can investigate this further? Thanks!

**Reply by mandrioli-twilio** [reply]
Hey @loreina,

Turns out this was happening because the project was changing `pnpm-lock.yaml` when running `pretty-quick` after commits. I've added `pnpm-lock.yaml` to `.prettierignore` and was able to run the project correctly.

**Reply by loreina** [reply, marked as answer]
Great, I'm glad the project is running correctly! If you run into other issues, feel free to make a new discussion.
